### DISCUSSION TITLE
Should we just remove mail.html

### DISCUSSION THREAD

**Discussion by mariocraft987** [cytrinc/bark-coding#269] [Admin General]
i dont think dumos gonna invite anyone to the api, or update it

**Comment by raynepaws** [comment]
> i dont think dumos gonna invite anyone to the api

why do you need an invite?

**Comment by mariocraft987** [comment]
small changes like adding `new Date()` to the "account creation"

**Reply by raynepaws** [reply]
> small changes like adding `new Date()` to the "account creation"

just make a fork, put changes there, and create a PR

**Reply by mariocraft987** [reply]
https://github.com/dumorando/bapiv2/pull/4

**Comment by dumorando** [comment]
> i dont think dumos gonna invite anyone to the api, or update it

yea probably not cuz im into c# now, i forgot a lot about javascript, the hosting provider sucks, and that code is some bs i made in 5 hours

and what even is mail.htm anyway

**Comment by mariocraft987** [comment]
notifcation page for new bark stuff ☠️
